package/kernel/linux/modules/netdevices.mk

```diff
@@ -1042,3 +1042,47 @@ define KernelPackage/be2net/description
 endef
 
 $(eval $(call KernelPackage,be2net))
+
+define KernelPackage/mlx4_core
```

Comment:
I would prefer `mlx4-core` here, as most kernel packages use hyphen in package name even if the module name contains underscore. [ex1](https://github.com/openwrt/openwrt/blob/master/package/kernel/linux/modules/netdevices.mk#L972) [ex2](https://github.com/openwrt/openwrt/blob/master/package/kernel/linux/modules/netdevices.mk#L1006)

Reply:
OK, but I want to look at the module configuration after the kernel 5.4, and also consider whether to set this driver to compile only for x86 targets (or arm?) After all, most architectures do not use the corresponding device